external-repos.repos

```diff
@@ -0,0 +1,61 @@
+repositories:
```

Comment:
Why is moveit2 not included here? This way travis just compiles the dependencies only.
Also we should rename this file to moveit2.repos for consistency.

Reply:
I think it would be cleaner to separate temporary dependencies to `AcutronicRobotics` and those that are not released yet somehow. Either just with comments or in a separate .repos file.

Reply:
> Why is moveit2 not included here? This way travis just compiles the dependencies only.
Also we should rename this file to moveit2.repos for consistency.

+1 for rename the file. I use in this way to mount the repo as a volume on Docker when we build it. This way we can build the code of the current branch. But, I can set the corresponding branch in moveit2.repos for each branch.

Reply:
I totally see the point in doing it this way. `moveit_ci` already handles the corresponding branch inside [travis.sh](https://github.com/ros-planning/moveit_ci/blob/master/travis.sh) anyway. I think the original reason why the parent repo was included in .rosinstall files is that it makes it easier to setup workspaces. @rhaschke any thoughts on this?

Reply:
I'm not fully sure what you are referring to exactly.
But, yes MoveIt's .rosinstall file comprises all repositories required to build MoveIt from source, i.e. also including MoveIt itself. travis.sh [here](https://github.com/ros-planning/moveit_ci/blob/e05280b6f127a35f2b40f0634fd7d9039a0f7e10/travis.sh#L185-L186) takes care that the to-be-tested repo is removed from the .rosinstall file before initializing the workspace.

Reply:
@LanderU Is there a reason to switch from `.rosinstall` to `.repos`? A good deal of the MoveIt infrastructure assumes the use of wstool with `.rosinstall` files. We can migrate over to vcs but we would probably want to have good reasons for doing so.

Reply:
We're using .repos because ROS2 is using this kind of file: https://github.com/ros2/ros2/blob/master/ros2.repos

Reply:
Does wstool + .rosinstall files not also work with ROS2? I think keeping the moveit1 approach, at least for now, will reduce a lot of headache. I think focusing on the minimum changes for ROS2 support would be the best first approach

Reply:
`wstool` still works since it's pretty much unrelated to any ROS/ROS2 functionality. `vcstools`  is a new alternative to `wstool` that is supported by OSRF and comes with some improved features (https://github.com/vcstools/wstool/issues/19#issuecomment-72942597) which we don't need for migration. However, the import function of `vcstools` supports .rosinstall files (see [README](https://github.com/dirk-thomas/vcstool)) so we should be able to use it as a drop-in replacement.